audio_fifo.go

```diff
@@ -0,0 +1,46 @@
+package astiav
+
+//#cgo pkg-config: libavutil
+//#include <libavutil/audio_fifo.h>
+//#include <stdlib.h>
+import "C"
+import "unsafe"
+
+type AudioFifo struct {
+	c *C.struct_AVAudioFifo
+}
+
+func newAudioFifoFromC(c *C.struct_AVAudioFifo) *AudioFifo {
+	if c == nil {
+		return nil
+	}
+	return &AudioFifo{c: c}
+}
+
+func AllocAudioFifo(sampleFmt SampleFormat, channels int, nbSamples int) *AudioFifo {
+	return newAudioFifoFromC(C.av_audio_fifo_alloc(C.enum_AVSampleFormat(sampleFmt), C.int(channels), C.int(nbSamples)))
+}
+
+func (a *AudioFifo) AudioFifoRealloc(nbSamples int) int {
+	return int(C.av_audio_fifo_realloc((*C.struct_AVAudioFifo)(a.c), C.int(nbSamples)))
```

Comment:
Can you replace all the `(*C.struct_AVAudioFifo)(a.c)` in this file with only `a.c`?